include/aspect/stokes_matrix_free.h

```diff
@@ -191,7 +191,7 @@ namespace aspect
         /**
          * Table which stores the viscosity on each quadrature point.
          */
-        Table<2, VectorizedArray<number> > one_over_viscosity;
+        Table<1, VectorizedArray<number> > one_over_viscosity;
```

Comment:
are these tables in the three operators the same? if yes, you could only compute and store it once and only keep a reference here.

Reply:
I think the issue is that the cells withing the matrix-free object can be iterated through in a different ordering depending on the matrix-free operator and so the indexing could change for each table.

Reply:
I guess we are now talking only about a couple of MB, so I am okay with it. I would be surprised if the order is different though?

Reply:
You are right, it is different for DG finite elements and continuous finite elements, but this isn't an issue for us since all operators are based on continuous elements.

I could add a function to each operator that sets a viscosity table?

Reply:
I say we don't worry right now with the reduction of the viscosity tables into 1. This whole structure will look different when we add in the higher order projection.

Reply:
ok, let's do that later.